docs/gradebook-canvas.html

```diff
@@ -3,5 +3,19 @@
 title: Format Canvas Gradebook
 ---
 
-<h1 class="mt-5">Format Canvas Gradebook Documentation</h1>
-<p>Follow documentation for this Feature</p>
+<h1 class="mt-5">Format Canvas Gradebook</h1>
+<p>This Feature allows you to format exported Canvas Gradebook CSV file for uploading into Faculty Center’s Grade Roster. </p>
+<h5>Who Can use Format Canvas Gradebook?</h5>
```

Comment:
```suggestion
<h2>Who Can use Format Canvas Gradebook?</h2>
```

Reply:
@pushyamig, @melindakraft is probably right about the heading levels, it's better for accessibility not to skip a bunch of them. If this is for style, you can use the `h2` and then apply an `"h5"` classname, I think. See https://getbootstrap.com/docs/4.0/content/typography/#headings